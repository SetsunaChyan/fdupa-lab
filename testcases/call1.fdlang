function main(){
    a = 20;
    call callee(a);
}

function callee(m){
    m = m + 1;
    check_interval(m, 21, 21);
}