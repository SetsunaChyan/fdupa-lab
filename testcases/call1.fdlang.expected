Line 8: YES; m in [21, 21]